scripts/update_robot.py

```diff
@@ -183,7 +184,55 @@ def on_shutdown():
     return nl.rc
 
 
+def rethink_ssh_update_version():
```

Comment:
Or maybe something like `ros_updateable_version()` - which is what the rest of the python script is about to do if the "check" passes.